src/Http/Uri.php

```diff
@@ -234,25 +235,15 @@ public function clone(array $props = [])
 
     /**
      * @param array $props
-     * @param bool $forwarded Deprecated! Todo: remove in 3.7.0
      * @return static
      */
-    public static function current(array $props = [], bool $forwarded = false)
+    public static function current(array $props = [])
     {
         if (static::$current !== null) {
             return static::$current;
         }
 
-        $uri = Server::requestUri();
-        $url = new static(array_merge([
-            'scheme' => Server::https() === true ? 'https' : 'http',
-            'host'   => Server::host(),
-            'port'   => Server::port(),
-            'path'   => $uri['path'],
-            'query'  => $uri['query'],
-        ], $props));
-
-        return $url;
+        return new static(App::instance()->url('current'), $props);
```

Comment:
I agree. We don't need to explicitly support such use cases, but I think we shouldn't throw rocks in the path if we don't need to. We use Kirby as a framework for the Demo Manager for example. Haven't tested it yet with this PR, but I believe it would break as well.

Reply:
Ok, I added a solution to keep things compatible.